Sources/UILocalNotification+Sweet.swift

```diff
@@ -0,0 +1,47 @@
+import UIKit
+
+public extension UILocalNotification {
+    static let idKey = "idKey"
+    static let fireDateKey = "fireDateKey"
+
+    public static func create(_ id: String, fireDate: Date, soundName: String? = nil, message: String, actionTitle: String? = nil) {
+        let notification = UILocalNotification()
+        notification.soundName = soundName
+
+        notification.fireDate = fireDate
+        notification.timeZone = TimeZone.current
+        notification.alertBody = message
+        notification.alertAction = actionTitle
```

Comment:
It doesn't make any difference, really. If it's `nil` Apple has some smart defaults for it.